KarhooUISDK/Extensions/UIView+Extensions.swift

```diff
@@ -166,10 +166,11 @@ extension UIView {
         }
     }
     
-    func addShadow(_ opacity: Float = Float(UIConstants.Alpha.shadow)) {
+    func addShadow(_ opacity: Float = Float(UIConstants.Alpha.shadow), radius: CGFloat = 3) {
```

Comment:
Why you used hardcoded value instead of some predefined constant?

Reply:
Added proper comment